data/favicon.json

```diff
@@ -188,6 +188,8 @@
 	"instagram.com": "instagram.com.ico",
 	"instantsystems.se": "instantsystems.se.png",
 	"intesasanpaolo.com": "intesasanpaolo.com.ico",
+	"inxmail.com": "inxmail.com.ico",
+	"inxserver.com": "inxmail.com.ico",
```

Comment:
Is this for mails what are coming from inxmail itself? Or for mails (e.g. newsletters) they send and sign on the behalf of overs?

At least for `inxserver.com` it is probably the latter.

And if it is the latter I do not want to add them to the favicon. I think signing newsletters just by the marketing company that is sending them is a confusing practice that should be discouraged from.

Reply:
It's for newsletters, they send. I'll remove it then...